### ISSUE TITLE
USBTOP no longer available in apt repos

### ISSUE THREAD

**Issue by 211217613** [aguinet/usbtop#38]
I'm testing in a VM and try to do a `sudo apt update` then a `sudo apt install usbtop` but the binary is not available.

```
vagrant@ubuntu-bionic:~$ sudo apt install usbtop
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package usbtop
```

I'm running bionic beaver
```
vagrant@ubuntu-bionic:~$ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.6 LTS"
```

**Comment by Luflosi** [comment]
According to https://repology.org/project/usbtop/versions, it is available in Ubuntu 20.04, 21.04, 21.10 and 22.04.

**Comment by 211217613** [comment]
```
vagrant@ubuntu-focal:~$ sudo apt search usbtop
Sorting... Done
Full Text Search... Done
usbtop/focal 1.0+dfsg-1build1 amd64
  utility to show bandwidth on USB buses and devices
```

Confirm that it is present on ubuntu focal 20.04.
